packages/sdk/test/vote.test.ts

```diff
@@ -1,18 +1,23 @@
 import { Token, Vote } from "../src";
+import { TokenImpl } from "../src/contracts/classes/token";
+import { VoteImpl } from "../src/contracts/classes/vote";
 import { sdk, signers } from "./hooks";
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 import { assert } from "chai";
 import { ethers } from "ethers";
-import { ethers as hardhatEthers } from "hardhat";
+import hardhat from "hardhat";
+
+// it's there, trust me bro
+const hardhatEthers = (hardhat as any).ethers;
```

Comment:
can we not just export it from the before all tests setup file? instead of doing this on every test file

Reply:
Probably would work, yeah